### PR TITLE
Arranged the server codes and routers.

### PR DESCRIPTION
**Pull request by dorhaimovich**
made some changes and moved functionalities into controllers from index.js

### CODE REVIEW

File: mongoConnection.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-const serverlogger = require(`./logger`);
-let logger = serverlogger.log;
+const serverLogger = require(`./logger`);
+let logger = serverLogger.log;
 const options = {
     useNewUrlParser: true,    // For deprecation warnings
     useUnifiedTopology: true // For deprecation warnings
 };
 
+console.log("here:::" + process.env.DB_HOST)
```
Comment (itay45977) [inline mongoConnection.js]: ah? remove it.

**Review by itay45977 — APPROVED** [review]
Remove the unnessesary line in mongoconnection line 9

File: mongoConnection.js (hunk 1 above)
Comment (alona26) [reply mongoConnection.js]: whyy

Comment (dorhaimovich) [reply mongoConnection.js]: that was for my debugging
Ill delete it now

**Pull request merged**